### PR TITLE
[TRANSLATION] Simplified Chinese translation improved.

### PR DESCRIPTION
**Pull request by njlyf2011**

### CODE REVIEW

**Comment by njlyf2011** [comment]
I restored some important Chinese folder names mentioned in SHELL32, USERENV and SYSSETUP to English, because the use of Chinese path can **cause** compatibility problems of some legacy Microsoft Windows programs in Chinese operation system.

Moreover, as a Chinese Windows user, it will be very unaccustomed to change some operating system paths to Chinese, because the Simplified Chinese Windows operation system **has not** translated these folder names into English.

File: dll/win32/newdev/lang/zh-CN.rc [hunk 1]
```diff
@@ -85,13 +86,13 @@ BEGIN
...
     LTEXT "单击完成关闭向导。", IDC_STATIC, 120, 174, 195, 12
-    LTEXT "未知的设备", IDC_DEVICE, 148, 53, 147, 12
+    LTEXT "未知设备", IDC_DEVICE, 148, 53, 147, 12
 END
 
 STRINGTABLE
 BEGIN
-    IDS_INSTALLWIZARDTITLE "新的硬件向导"
-    IDS_UPDATEWIZARDTITLE "更新驱动程序向导"
+    IDS_INSTALLWIZARDTITLE "找到新的硬件向导""
```
Comment (learn-more) [inline dll/win32/newdev/lang/zh-CN.rc]: You have a double `"` at the end of this text, which makes the build fail.

Comment (njlyf2011) [reply dll/win32/newdev/lang/zh-CN.rc]: Corrected.

**Comment by HBelusca** [comment]
@learn-more : your requested change has been applied. I'm going to commit the translations.

**Pull request merged**